index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
```

Comment:
```suggestion
Prior to verification when using the JOSE or COSE specifications to secure a
<a>verifiable credential</a> or <a>verifiable presentation</a>.
```

Reply:
There is going to be better language here - I would acually go so far as to say that if you are using JOSE/COSE you don't look at the payload as json or json-ld until verification has taken place.  this means of course that you don't process as json-ld prior to verification.

Reply:
then if you go further with the protected header post verification, and `cty` tells you this thing you just verified is JSON-LD, e.g. via `vc+ld+json`, then you go through and have an option to process according to the content type, so you can process as `ld+json` (and get the glorious semantics` or just as `json` for some other reason (as you noted below re media types).

Reply:
There is a confusion between payload and securing mechanism processing. JSON-LD is needed for the payload, for securing mechanism, JWS library might be enough.